rules/linux/defense_evasion_sus_utility_executed_via_tmux_or_screen.toml

```diff
@@ -29,26 +29,23 @@ tags = [
 ]
 timestamp_override = "event.ingested"
 type = "eql"
-
 query = '''
 process where host.os.type == "linux" and event.type == "start" and event.action in ("exec", "exec_event") and 
-process.parent.name in ("screen", "tmux") and process.name : (
-  "nmap", "nc", "ncat", "netcat", "socat", "nc.openbsd", "ngrok", "ping", "java", "python*", "php*", "perl", "ruby",
-  "lua*", "openssl", "telnet", "awk", "wget", "curl", "id"
-  )
+process.parent.name in ("screen", "tmux") and process.name like (
```

Comment:
@Aegrah did we test the use of this in Kibana EQL? I assume it's built from the same grammar so should still be feasible as endpoint, but just wanted to be sure as they are different in some capabilities.

Reply:
I just checked, and it works. But good call!